### ISSUE TITLE
Design Intent - Accessibility Feedback - MHV Medications Prescription Grouping

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#95931]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Very happy to see this work! I think this does a lot to improve cognitive load, really appreciate it being prioritized.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Must:** To the extent that it's possible, make sure the links for each card are unique. I made a comment about this in the meeting without totally thinking things through, however. If two links go to the same destination, it's okay for them to have the same link text --- meaning, if an active `[medication name] [dosage]` and an inactive/expired/some other status `[medication name] [dosage]` have links that go to the same details page, that's okay for the link text to be redundant. If the links are going to different places, that's when you'll want to be careful about keeping them unique.
   
   Either way, my advice to work with @BobbyBaileyRB and other accessibility folks still stands!
- [ ] **Consider:** There's some evidence from research that having the card heading also be a link can cause some trouble for screen reader users. Not a must at this point, but consider exploring ways of structuring the card so that you have a separate link rather than a linked heading.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone
